app/src/main/java/com/orange/ods/app/ui/components/sliders/ComponentSliders.kt

```diff
@@ -89,8 +89,8 @@ fun ComponentSliders() {
                         steps = steps,
                         valueRange = valueRange,
                         onValueChange = { sliderPosition = it },
-                        leftIcon = leftIcon,
-                        rightIcon = rightIcon
+                        startIcon = leftIcon,
+                        endIcon = rightIcon
```

Comment:
We should rename `leftIcon`, `leftIconContentDescription`, `rightIcon` and `rightIconContentDescription` variables too.

And also code implementation:

```kotlin
classInstance<OdsSliderIcon>("leftIcon") {
    painter()
    contentDescription(leftIconContentDescription)
}
classInstance<OdsSliderIcon>("rightIcon") {
    painter()
    contentDescription(rightIconContentDescription)
}
```

Reply:
Damn I missed them!